src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -369,27 +339,12 @@ private static FlutterSdk getFlutterFromDartSdkLibrary(Library lib, VirtualFile
   }
 
   public enum Command {
-
-    CREATE("Flutter create", "create") {
-      @Override
-      void onStart(@Nullable Module module, @Nullable VirtualFile workingDir, @NotNull String... args) {
-        // Enable Dart.
-        if (module != null) {
-          ApplicationManager.getApplication()
-            .invokeLater(() -> ApplicationManager.getApplication().runWriteAction(() -> DartPlugin.enableDartSdk(module)));
-        }
-      }
-    },
+    CREATE("Flutter create", "create"),
     DOCTOR("Flutter doctor", "doctor"),
     PACKAGES_GET("Flutter packages get", "packages", "get"),
     PACKAGES_UPGRADE("Flutter packages upgrade", "packages", "upgrade"),
     UPGRADE("Flutter upgrade", "upgrade"),
-    VERSION("Flutter version", "--version") {
-      @Override
-      boolean attachToConsole() {
-        return false;
-      }
-    };
```

Comment:
I looked at the usages of VERSION and there aren't any that pass a Project to startProcessWithoutModule(). That's enough to prevent a console from being attached, so this is basically dead code.